### DISCUSSION TITLE
Just a couple of questions.

### DISCUSSION THREAD

**Discussion by Stanlyhalo** [Chlumsky/msdfgen#168] [General]
Hey, I'm trying to use this, and I have it save a character as a png, not sure if it's suppose to have different color bands, I'm assuming that's normal. But my main questions are simply: Is there a way to just loop through the glyphs and get their correct unicode for proper usage? And is there a way to get ligatures?

I'm okay with making a tool in my project to save a custom font format and can add the ligatures and assign the glyphs manually, but I was just wondering.

Edit*: I could also just add a regex like system to get the glyphs I need.

**Comment by Chlumsky** [comment]
Glyphs (including ligatures) can be accessed by their font-specific indices in the command line version as `-font x.ttf g123` (`g` prefix before index). This project does not provide a way to get their Unicode index (and as you know, not all glyphs map to a Unicode character), but you can instead access them by their Unicode index without the `g` prefix. Please see [msdf-atlas-gen](https://github.com/Chlumsky/msdf-atlas-gen) and check if it helps your use case in any way. You may also consider linking msdfgen as a library and fetching additional font data from FreeType or HarfBuzz before having the desired distance fields generated with a bit of custom code.

**Reply by Stanlyhalo** [reply]
Alright, thanks for the information, I'll check it out, see what I can do. If it comes down to adding a bit of custom code, I don't mind. This api is already pretty great.
